app/views/concerns/admin/sortable/_sort_options.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if defined?(sort_options) && sort_options %>
```

Comment:
Do we really need this conditional?

Reply:
It prevents the page breaking if no sort options are defined at controller level, but where this view gets rendered from elsewhere. Might be able to drop it / work around it not sure.